run-tests.php

```diff
@@ -1930,7 +1932,7 @@ function run_test($php, $file, $env)
 	if (array_key_exists('CGI', $section_text) || !empty($section_text['GET']) || !empty($section_text['POST']) || !empty($section_text['GZIP_POST']) || !empty($section_text['DEFLATE_POST']) || !empty($section_text['POST_RAW']) || !empty($section_text['PUT']) || !empty($section_text['COOKIE']) || !empty($section_text['EXPECTHEADERS'])) {
 		if (isset($php_cgi)) {
 			$php = $php_cgi . ' -C ';
-		} else if (!strncasecmp(PHP_OS, "win", 3) && file_exists(dirname($php) . "/php-cgi.exe")) {
```

Comment:
I suppose the usage of ``strncasecmp()`` is irrelevant compared to ``substr()`` but it is weird seeing it.